workflow/context.go

```diff
@@ -0,0 +1,71 @@
+package workflow
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/microsoft/durabletask-go/task"
+)
+
+type Context struct {
```

Comment:
Should this be named `WorkflowContext` to avoid collisions or confusion with `context.Context`?